modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -3,6 +3,8 @@
 	var/is_shifted
 	var/shifting //If we are in the shifting setting.
 
+	var/passthroughable = 0
```

Comment:
Also, I'd like for you to add some documentation to this variable, with `/// Here's what this is used for.` above the variable (insert your own text of course)
```suggestion
	var/passthroughable = NONE
```